package-configuration/linux-cli/get-cloudify.py

```diff
@@ -109,6 +109,7 @@ def _print_proc_out(proc):
     stdout_line = proc.stdout.readline()
     if len(stdout_line) > 0:
         prn('STDOUT: {0}'.format(stdout_line))
+    return stdout_line
```

Comment:
now this function does two things (printing + returning) instead of one. this breaks the good old "function should either do something or tell you something".